client/src/containers/metadata/index.tsx

```diff
@@ -9,13 +9,14 @@ import { DialogDescription, DialogHeader, DialogTitle } from '@/components/ui/di
 export default function Metadata(layer: LayerTyped) {
   const { title, metadata } = layer;
   const { description = '' } = metadata;
+  const descriptionString = typeof description === 'string' ? description : '';
 
   return (
     <article className="max-h-[90svh] overflow-y-auto overflow-x-hidden p-6">
       <DialogHeader>
         <DialogTitle className="text-3xl">{title}</DialogTitle>
         <DialogDescription>
-          <ReactMarkdown className="prose prose-sm prose-slate">{description}</ReactMarkdown>
+          <ReactMarkdown className="prose prose-sm prose-slate">{descriptionString}</ReactMarkdown>
```

Comment:
We don't have an info button in the legend, so I don't think this component is needed anymore. We also don't use Markdown.